packages/core/src/Cardano/util/numeric.ts

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable no-use-before-define */
+
+export declare type Ratio = number;
```

Comment:
not really, I just kept it from Ogmios types, actually I am a bit confused by the over aliasing I am seeing all over the place, but I am not sure what should be the good practice regarding this or what the guideline so I am just keeping it as it is, this is the first language where I see this is done, usually the field name plus the documentation is enough, in some cases aliasing makes sense, but most of the time to me look superfluous ( stuff like { epochLengh: EpochLenght } vs simply just { epochLengh: number}), I would lean to just use primitive types and document properly.

Reply:
Domain-type aliases are valuable since they guard against passing invalid values that may happen to match primitive types, and it also improves documentation since a single type alias' documentation is then inherited by all consumers.

Reply:
Types that use OpaqueString guards against passing invalid values. Type aliases don't, do they :thinking:? I find it helpful for documentation, but doc comments are even more helpful as they show up in 'intellisense' popup

Reply:
> Type aliases don't, do they ?

Yikes, that's surprising:
https://www.typescriptlang.org/play?#code/C4TwDgpgBAYg9nKBeKA7ArgWwEYQE4DcAsAFCiRQBCAhnsmlroaaQMZyoDOwUY1nneIhQAKAGYIAXLAQBKZAD4o7LnAA2EAHRq4Ac3FziJNh25RstaTTooAjEb4ChIi3lkEgA

OK, scrap that aspect. I still think it's worth doing for centralising documentation. Can we not configure IDEs to access type doc blocks?

Reply:
> Can we not configure IDEs to access type doc blocks?

Maybe, I think it comes from TypeScript tooling that IDE then utilizes

Reply:
Also IMHO I think the documentation should also be available where the type, structure or function prototype is defined, some ppl uses simple dev environments (IE vi or emacs), and I think even if I open a source file with a text editor the relevant documentation should be available without that need to go hunting for the documentation in different source files

Reply:
I.E I would spect all the relevant documentation regarding a function or class is located at where that function or class is defined

Reply:
Yep, everything that is exported should have doc comments

> some ppl uses simple dev environments (IE vi or emacs)

Those have plugins that provide tooling integration too, but I know what you're saying.

Reply:
addressed here: https://github.com/input-output-hk/cardano-js-sdk/pull/388/commits/13a4d6b921fa14faf0c75a591a3805de1d3c13a5